05_book_titles/book.rb

```diff
@@ -1,3 +1,77 @@
 class Book
 # write your code here
+    # attr_writer :title
+
+    # def initialize
+    #     @title = title
+    # end
+
+    def title
+        titleize
+    end
+
+    def title=(title)
+        @title = title
+    end
+
+    def titleize
+        array = @title.split
+        array2 = []
+        i = 0
+        
+        while i < array.length
+            if i == 0
+                array2 << array[i].capitalize
```

Comment:
We can do similar logic for simon says program,  initialize a variable something similar to  `words=["a","and","the","over"]` and then use the keyword `little_words.include?` to filter out certain words